tests/std/tests/GH_000639_nvcc_include_all/env.lst

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+PM_COMPILER="nvcc" PM_CL="--x cu -Xcompiler -Od,-EHsc,-nologo,-W4,-WX,-openmp"
+RUNALL_CROSSLIST
+PM_CL="-Xcompiler -MT"
+PM_CL="--debug -Xcompiler -MTd,-D_ITERATOR_DEBUG_LEVEL=2"

```

Comment:
I think `-D_ITERATOR_DEBUG_LEVEL=2` is unnecessary here. In the usual_matrix etc. files, we vary IDL, so it's simplest to always specify it (even when it happens to match the default setting). But for configurations where we always use the default, omitting it is simpler - and indeed `-MT` above omits IDL=0.